typescript/cli/src/send/message.ts

```diff
@@ -106,19 +115,29 @@ async function executeDelivery({
     const formattedRecipient = addressToBytes32(recipient);
 
     log('Getting gas quote');
-    const value = await mailbox['quoteDispatch(uint32,bytes32,bytes)'](
+    const value = await mailbox[
+      'quoteDispatch(uint32,bytes32,bytes,bytes,address)'
+    ](
       destinationDomain,
       formattedRecipient,
       MESSAGE_BODY,
+      ethers.utils.hexlify([]),
+      hook,
     );
     log(`Paying for gas with ${value} wei`);
 
     log('Dispatching message');
-    const messageTx = await mailbox['dispatch(uint32,bytes32,bytes)'](
+    const messageTx = await mailbox[
+      'dispatch(uint32,bytes32,bytes,bytes,address)'
+    ](
       destinationDomain,
       formattedRecipient,
       MESSAGE_BODY,
```

Comment:
Completely optional but someone recently asked for the ability to set custom content for the MESSAGE_BODY here (like via a CLI arg). If you feel like adding that on go ahead, otherwise np I'll get to it later

Reply:
seems easy and useful for testing :)

Reply:
Done!